docs/self-managed/components/modeler/web-modeler/configuration/configuration.md

```diff
@@ -24,32 +24,56 @@ Clusters must be configured using the following options to access the cluster fr
-| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_AUDIENCE_ZEEBE`    | The OAuth token audience for Zeebe. Only required for authentication type `CLIENT_CREDENTIALS`.                | `zeebe-api`                                                                 |
-| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_AUDIENCE_OPERATE`  | The OAuth token audience for Operate. Only required for authentication type `CLIENT_CREDENTIALS`.              | `operate-api`                                                               |
-| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_AUDIENCE_TASKLIST` | The OAuth token audience for Tasklist. Only required for authentication type `CLIENT_CREDENTIALS`.             | `tasklist-api`                                                              |
+To add additional clusters, increment the `0` value for each variable (e.g. `CAMUNDA_MODELER_CLUSTERS_1_NAME`).
+
+:::info Cluster version
+The available configuration options depend on the version of the cluster:
+
+- [Common configuration (all cluster versions)](#common-configuration-all-cluster-versions)
+- [Additional configuration for cluster versions >= 8.8](#additional-configuration-for-cluster-versions--88)
+- [Additional configuration for cluster versions < 8.8](#additional-configuration-for-cluster-versions--88-1)
+  :::
+
+#### Common configuration (all cluster versions)
+
+| Environment variable                        | Description                                                                                   | Example value                    |
+| ------------------------------------------- | --------------------------------------------------------------------------------------------- | -------------------------------- |
+| `CAMUNDA_MODELER_CLUSTERS_0_ID`             | A unique identifier to use for your cluster.                                                  | `test-cluster-1`                 |
+| `CAMUNDA_MODELER_CLUSTERS_0_NAME`           | The name of your cluster.                                                                     | `Test Cluster 1`                 |
+| `CAMUNDA_MODELER_CLUSTERS_0_VERSION`        | The Camunda version used by this cluster.                                                     | `8.8.0`                          |
+| `CAMUNDA_MODELER_CLUSTERS_0_AUTHENTICATION` | The [authentication](#available-authentication-methods) to use with your cluster.             | `BEARER_TOKEN`                   |
+| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_URL`      | The address of your token issuer. Only required for authentication type `CLIENT_CREDENTIALS`. | `https://auth.example.com/token` |
+| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_SCOPE`    | [optional]<br/>A list of the scopes to use with this cluster separated by spaces.             | `openid email profile`           |
```

Comment:
Maybe we can also move these two to the "< 8.8" section to keep all the `OAUTH_*` variables together, even though `CLIENT_CREDENTIALS` authentication would also be supported with an 8.8 cluster.